openquake/server/static/js/engine.js

```diff
@@ -148,30 +170,149 @@
                 var myXhr = $.ajax({url: gem_oq_server_url + "/v1/calc/" + calc_id + "/traceback",
                                     error: function (jqXHR, textStatus, errorThrown) {
                                         if (jqXHR.status == 404) {
-                                            diaerror.showDiaError("Calculation " + calc_id + " not found.");
+                                            diaerror.show("Calculation " + calc_id + " not found.");
                                         }
                                         else {
-                                            diaerror.showDiaError("Error retrieving traceback for calculation " + calc_id + ".", textStatus);
+                                            diaerror.show("Error retrieving traceback for calculation " + calc_id + ".", textStatus);
                                         }
                                         // alert("Error: " + textStatus);
```

Comment:
Even if it is not part of this PR we should remote it or at least chage it to `console.log` and keep the comment